prow/plugins/config.go

```diff
@@ -364,6 +365,12 @@ type Milestone struct {
 	MaintainersFriendlyName string `json:"maintainers_friendly_name,omitempty"`
 }
 
+// MilestoneApplier contains the configuration for the milestoneapplier plugin.
+type MilestoneApplier struct {
+	// BranchToMilestone is a map of branch to the current milestone for that branch.
+	BranchToMilestone map[string]string `json:"branch_to_milestone,omitempty"`
```

Comment:
nit: might be cleaner in config if this were `milestones` or inlined or something

```yaml
milestone_applier:
  kubernetes/kubernetes:
    master: v1.15
```

```yaml
milestone_applier:
  kubernetes/kubernetes:
    milestones:
      master: v1.15
```

Reply:
Replaced this with a `map[string]map[string]string`.

This makes the config cleaner/simpler, but I'm concerned that we might not be able to extend the config for `MilestoneApplier` in the future. Not sure if this is concerning for this plugin or not though...